sdk/eventgrid/Microsoft.Azure.EventGrid/src/Customization/EventTypes.cs

```diff
@@ -85,5 +85,7 @@ public class EventTypes
         // Storage events
         public const string StorageBlobCreatedEvent = "Microsoft.Storage.BlobCreated";
         public const string StorageBlobDeletedEvent = "Microsoft.Storage.BlobDeleted";
+        public const string StorageDirectoryCreatedEvent = "Microsoft.Storage.DirectoryCreated";
```

Comment:
Don't you need to add the mappings (in this file and in SystemEventTypeMappings.cs) for StorageBlobRenamedEvent and StorageDirectoryRenamedEvent as well?

Reply:
Sorry about this mistake. I've updated with StorageBlobRenamedEvent and StorageDirectoryRenamedEvent.